model/host/host.go

```diff
@@ -475,6 +475,10 @@ func (h *Host) IsEphemeral() bool {
 	return utility.StringSliceContains(evergreen.ProviderSpawnable, h.Provider)
 }
 
+func (h *Host) IsStatic() bool {
```

Comment:
This functions is equivalent to `j.host.IsEphemeral`. I also don't think we need to worry about future-proofing that much since there's no real plans to introduce some third kind of cloud provider that's neither static nor ephemeral.